### DISCUSSION TITLE
Possible to export/import images/files?

### DISCUSSION THREAD

**Discussion by goaliedude3919** [sysadminsmedia/homebox#127] [General]
I'm migrating my Homebox instance to a new server and also migrating to this new fork. I have a lot of PDF files and images assigned to items, but I don't see any way of exporting those to bring over to the new instance, as the only export option I see produces a TSV file. Am I going to have to do this manually?

**Comment by tankerkiller125** [comment]
For migrating to the new fork you don't have to do anything other than change the Docker image/binary your using. It should take in your existing database and just immediately start using it.

Ideally for a move you would backup your existing install and it's data storage volume/location and then restore the data on the new server.

**Comment by weado** [comment]
Hi,
I've just discovered your project, and I think it's great.
Is it possible to upload PDF datasheet for item ?
